src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,67 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        Manufacturer mercedes = new Manufacturer(null, "Mercedes", "Germany");
+        Manufacturer lada = new Manufacturer(null, "Lada", "Ukraine");
+        Manufacturer mazda = new Manufacturer(null, "Mazda", "Japan");
+        manufacturerService.create(mercedes);
+        manufacturerService.create(lada);
+        manufacturerService.create(mazda);
+
+        DriverService driverService = (DriverService) injector
+                .getInstance(DriverService.class);
+        Driver yaroslav = new Driver(null, "Yaroslav", "01234");
+        Driver eugene = new Driver(null, "Eugene", "02344");
+        Driver volodimir = new Driver(null, "Volodimir", "03454");
```

Comment:
passing of nulls in constructors is a bad practice, create new constructors where you won't need to pass them